### PR TITLE
Replaces the two chairs in perma with bolted ones

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request
As title

# Explain why it's good for the game
Less perma prisoners chairing themselves to death

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
maptweak: Changed the chairs in perma cells to bolted ones.
amptweak: Changed the chairs in Interrogation to bolted ones.
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
im pretty sure you're not supposed to open prs based on a current round's events

also, why remove the literal only form of stimulation perma prisoners have

**Comment by NewyearnewmeUwu** [comment]
dont get put in solitary then

File: maps/map_files/USS_Almayer/USS_Almayer.dmm [hunk 1]
```diff
@@ -63361,7 +63363,9 @@
 /turf/open/floor/almayer,
 /area/almayer/living/briefing)
 "tnb" = (
-/obj/structure/bed/chair/comfy/black,
+/obj/structure/bed/chair/comfy/black{
+	dir = 4
```
Comment (morrowwolf) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:63367]: Randomly rotated chair in port railgun/missile room.

File: maps/map_files/USS_Almayer/USS_Almayer.dmm [hunk 2]
```diff
@@ -31120,7 +31120,9 @@
 	},
 /area/almayer/living/captain_mess)
 "dOL" = (
-/obj/structure/bed/chair/comfy/black,
+/obj/structure/bed/chair/comfy/black{
+	dir = 4
```
Comment (morrowwolf) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:31124]: Random turned chair in CMP's office.

File: maps/map_files/USS_Almayer/USS_Almayer.dmm [hunk 3]
```diff
@@ -42781,7 +42783,7 @@
 /obj/structure/machinery/light{
 	dir = 1
 	},
-/obj/structure/bed/chair{
+/obj/structure/bed/chair/bolted{
```
Comment (morrowwolf) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:42786]: I count six bolted changes when there's only two chairs in perma. Please log locations and changes in changelog.

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: maps/map_files/USS_Almayer/USS_Almayer.dmm (hunk 1 above)
Comment (realforest2001) [reply maps/map_files/USS_Almayer/USS_Almayer.dmm:63367]: No idea why these rotated.

**Review by morrowwolf — APPROVED** [review]

**Pull request merged**